docker/dockerfiles/artifacts/artifact.Dockerfile

```diff
@@ -51,4 +52,14 @@ COPY --from=fe-builder /build/starrocks/output /release/fe_artifacts
 COPY --from=be-builder /build/starrocks/output /release/be_artifacts
 COPY --from=broker-builder /build/starrocks/fs_brokers/apache_hdfs_broker/output /release/broker_artifacts
 
+# download the latest dd-java-agent
+ADD 'https://dtdg.co/latest-java-tracer' /release/fe_artifacts/fe/datadog/dd-java-agent.jar
+
+# Get ddprof for BE profiling
+ADD https://github.com/DataDog/ddprof/releases/download/v0.15.3/ddprof-0.15.3-${TARGETARCH}-linux.tar.xz ddprof-linux.tar.xz
```

Comment:
This seems the only workable solution, the other two have issues as described below:
* It's not able to set ENV during docker image build, see https://github.com/moby/moby/issues/29110;
* `wget` in `busybox` doesn't work in multi-arc, see https://github.com/docker-library/busybox/issues/162

Reply:
if it is the busybox causes trouble, you can add a new build stage with a different image base to download the necessary binaries, copy the binary in busybox stage from the download stage.

Reply:
good point, let me try with another base image like `ubuntu:xxx`